cmd/neofs-cli/internal/client/sdk.go

```diff
@@ -69,7 +69,7 @@ func GetSDKClient(key *ecdsa.PrivateKey, addr network.Address) (*client.Client,
 }
 
 // GetCurrentEpoch returns current epoch.
-func GetCurrentEpoch(ctx context.Context, endpoint string) (uint64, error) {
+func GetCurrentEpoch(cmd *cobra.Command, ctx context.Context, endpoint string) (uint64, error) {
```

Comment:
`revive` linter is not happy about this change.

Reply:
Thanks, updated.